contrib/python/pypodman/pypodman/lib/parser_actions.py

```diff
@@ -63,6 +64,53 @@ def __call__(self, parser, namespace, values, option_string=None):
             setattr(namespace, self.dest, val)
 
 
+class ChangeAction(argparse.Action):
+    """Convert and validate change argument."""
+
+    def __init__(self,
+                 option_strings,
+                 dest,
+                 nargs=None,
+                 const=None,
+                 default=None,
+                 type=None,
+                 choices=None,
+                 required=False,
+                 help=None,
+                 metavar='OPT=VALUE'):
+        """Create ChangeAction object."""
+        help = (help or '') + ('Apply change(s) to the new image.'
+                               ' May be given multiple times.')
+
+        super().__init__(
+            option_strings=option_strings,
+            dest=dest,
+            nargs=nargs,
+            const=const,
+            default=default,
+            type=type,
+            choices=choices,
+            required=required,
+            help=help,
+            metavar=metavar)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        """Convert and Validate input."""
+        items = getattr(namespace, self.dest, None) or {}
+        items = copy.copy(items)
+
+        choices = ('CMD', 'ENTRYPOINT', 'ENV', 'EXPOSE', 'LABEL', 'ONBUILD',
+                   'STOPSIGNAL', 'USER', 'VOLUME', 'WORKDIR')
+
```

Comment:
Is this supposed to emulate the functionality in `buildah config`?  If so, there are a number of other choices that we could potentially add. https://github.com/containers/buildah/blob/master/docs/buildah-config.md

Reply:
@TomSweeneyRedHat he's just moving them from ``commit_action.py`` (above), nothing new here, only consolidated and made modular.